docs/Configure-a-Project/Project-Commands.md

```diff
@@ -745,14 +740,6 @@ Parameters:
     missing/empty will run for all
 -   `working_directory`: local path to the working directory
 
-## manifest.load
```

Comment:
We should keep this right?

Reply:
I don't think so, because all manifest.load does is let the user know that it's deprecated, so there's nothing to test. 
We are only keeping it in the registry in order for it to not result in errors for everyone who hasn't removed it yet.

Reply:
(this probably should have been removed when the command was first deprecated)